src/document/position.js

```diff
@@ -113,11 +135,21 @@ CKEDITOR.define( [ 'utils', 'ckeditorerror' ], ( utils, CKEditorError ) => {
 		 *
 		 * @returns {Number[]} Path to the parent.
 		 */
-		getParentPath() {
```

Comment:
I thought a bit about it and commented on that issue. According to my comment, this should be a normal method, not a getter.